### PR TITLE
added sidebar drawer and other stuffs

### PR DESCRIPTION
**Pull request by vigneshpy**

### CODE REVIEW

**Comment by vigneshpy** [comment]
Sample image 
Sidebar

File: App.tsx [hunk 1]
```diff
@@ -1,21 +1,24 @@
-import { StatusBar } from 'expo-status-bar';
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import * as React from 'react';
```
Comment (ajwebdev) [inline App.tsx:1]: please import what ever you want. dont import all because it takes more time to complie

File: App.tsx [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
+import {StyleSheet,View } from 'react-native';
+import { createDrawerNavigator } from '@react-navigation/drawer';
+import { NavigationContainer } from '@react-navigation/native';
+import CustomDrawer from  './Components/Drawer/Drawer';
+import AppHeader from  './Components/Header/AppHeader';
+
+const styles = StyleSheet.create({
+  Container: { flex: 1, alignItems: 'center', justifyContent: 'center' },
+
```
Comment (ajwebdev) [inline App.tsx:10]: avoid whitespaces, use prettier in your editor

File: Components/Drawer/DrawerContent.tsx [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+              <Image
+                source={require( "../../assets/hello-mobiles.png" )}
+                style={{ width:300, height: 100 }}
+              />
+            
+            </View>
+          </View>
+
+          <DrawerItem
+            // icon={({ color, size }) => (
```
Comment (ajwebdev) [inline Components/Drawer/DrawerContent.tsx:31]: remove comments

File: Components/Drawer/DrawerToggler.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+const DrawerToggler = (props)=> {
+    //Structure for the navigatin Drawer
+    const toggleDrawer = () => {
+      //Props to open/close the drawer
+      props.navigationProps.toggleDrawer();
+    };
+  
+    return (
+      <View style={{ flexDirection: 'row' }}>
+        <TouchableOpacity onPress={()=> toggleDrawer()}>
```
Comment (ajwebdev) [inline Components/Drawer/DrawerToggler.tsx:14]: change this to onPress={ toggleDrawer}

File: Components/Drawer/DrawerContent.tsx [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+            //   <Icon
+            //     name="home-outline"
+            //     color={color}
+            //     size={size}
+            //   />
+            // )}
+            label="Home" style={styles.label}
+            onPress={() => {
+              props.navigation.navigate("DashBoard");
```
Comment (ajwebdev) [inline Components/Drawer/DrawerContent.tsx:40]: change this into one common function

File: Components/Drawer/DrawerContent.tsx [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+            }}
+          />
+          <DrawerItem
+            // icon={({ color, size }) => (
+            //   <Icon
+            //     name="account-outline"
```
Comment (ajwebdev) [inline Components/Drawer/DrawerContent.tsx:46]: create a common component like customDrawerItem

File: Components/Screens/Accessories/AccessoriesScreen.tsx [hunk 7]
```diff
@@ -0,0 +1,12 @@
+import  React from 'react';
+import { Button, View, Text} from 'react-native';
+
+
+export default function AccessoriesScreen({ navigation }) {
+    console.log(navigation);
```
Comment (ajwebdev) [inline Components/Screens/Accessories/AccessoriesScreen.tsx:6]: remove log

**Pull request merged**